cmd/func/main.go

```diff
@@ -39,6 +42,26 @@ func main() {
 		if ctx.Err() != nil {
 			os.Exit(130)
 		}
+
+		if client.IsErrConnectionFailed(err) && !dockerOrPodmanInstalled() {
+			fmt.Fprintln(os.Stderr, `Docker/Podman not available.
```

Comment:
Maybe? It's tricky, because there are two things going on here. There is no local binary to start, but also we can't connect - which does not have to be local. The user could resolve this issue as well by setting `DOCKER_HOST` correctly, couldn't they?

Reply:
If we cannot connect AND docker nor podman is present then 99.9% of time user should install it.

This may result in false positive only for power user connecting to SSH, and they should know what they are doing.

Reply:
Also I am trying best to make it work without DOCKER_HOST.